packages/payment-processor/src/payment/batch-proxy.ts

```diff
@@ -116,9 +116,7 @@ export function encodePayBatchRequest(requests: ClientTypes.IRequestData[]): str
     const pn = getPaymentNetworkExtension(requests[0]);
     for (let i = 0; i < requests.length; i++) {
       validateErc20FeeProxyRequest(requests[i]);
-      if (!comparePnTypeAndVersion(pn, requests[i])) {
-        throw new Error(`Every payment network type and version must be identical`);
-      }
+      comparePnTypeAndVersion(pn, requests[i]);
```

Comment:
are you sure changing how this function manages errors is not breaking something in other projects ?

Reply:
Yes, I had created this function and I am the only one to use it